src/components/resume/Header.vue

```diff
@@ -14,17 +14,18 @@ const name = computed(() => {
 <template>
   <div v-if="resume">
     <div v-if="resume.basics.avatar" class="flex justify-between items-center">
-      <span class="font-normal text-center text-4xl">
-        {{ name || 'Your Name' }}
-      </span>
-      <img :src="resume.basics.avatar" class="max-h-3cm max-w-3cm ">
+      <div class="font-normal flex flex-col gap-2">
+        <span class="text-4xl">{{ name || 'Your Name' }}</span>
```

Comment:
Why not show the label when has avatar? 😆

![image](https://github.com/user-attachments/assets/9d77016a-a7ee-4ead-bdfd-8c1981c166f9)